packages/react-image/src/components/Image/Image.scss

```diff
@@ -6,12 +6,15 @@
  * they will be resolved on the :root element only; not at usage time.
 **/
 :root {
-  --image-borderRadius: 0;
+  --image-borderColor: rgb(225, 223, 221) /* TODO: replace with a proper value */;
+  --image-borderRadius: none;
+  --imageBoxShadhow: 0px 2px 4px -0.75px rgba(0, 0, 0, 0.1) /* --effects-elevation4 */;
+
+  --image-rounded-small-borderWidth: 2px /* --effects-roundedCorner2 */;
+  --image-rounded-medium-borderWidth: 4px /* --effects-roundedCorner4 */;
+  --image-rounded-large-borderWidth: 6px /* --effects-roundedCorner6 */;
```

Comment:
@dzearing is there some guidance how tokens should be named and how they are connected with themes?

Reply:
Does it make sense to make an image specific set of border tokens? I could understand a bordered modifier assigning values to general component tokens. This is probably more of a designer question; i don't understand the use case well.

Component tokens will generally follow the form:

```
--{componentName}-{state if applicable}-{property}
```

For flags, what you're doing here seems logical; you're defining what the enum `rounded` when set to `small` translates to for `borderWidth`. But I would question if that is valuable for a component token. Are there many scenarios where we would only want to round the border, and not apply some other effect? Would a variant make more sense?

How they fit into the theme - this would work right now:

```
const theme = { 
  tokens: {
   color: {},
   effects: {
      roundedCorner2: '2px'
   },
   etc: {},

    image: { // component tokens
      rounded: {
         small: {
           // we should have this string in a consts in a base-theme package
           borderWidth: 'var(--effects-roundedCorner2)',
        }
    }
 }
};
```

On type safety, the image object could be case as an `ImageTokens` object, or, we standardize on a factory to produce the image `createImageTheme({ partial image token })`.